packages/experiments/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -0,0 +1,234 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  css,
+  getId,
+  autobind,
+  customizable
+} from 'office-ui-fabric-react/lib/Utilities';
+import {
+  ICommandBar,
+  ICommandBarProps,
+  ICommandBarItemProps,
+  ICommandBarStyleProps,
+  ICommandBarStyles
+} from './CommandBar.types';
+import { CommandBarButton } from 'office-ui-fabric-react/lib/Button';
+import { OverflowSet, IOverflowSet } from 'office-ui-fabric-react/lib/OverflowSet';
+import { ResizeGroup } from 'office-ui-fabric-react/lib/ResizeGroup';
+import { TooltipHost } from 'office-ui-fabric-react/lib/Tooltip';
+import {
+  classNamesFunction
```

Comment:
This will break now. Can you sync from master and import this from utilities?

Reply:
fixed